ftplugin/latex-suite/bibtools.py

```diff
@@ -27,8 +33,12 @@ def __init__(self, txt, macros={}):
         """
 
         if macros:
-            for k, v in macros.iteritems():
-                txt = txt.replace(k, '{' + v + '}')
+            try:
+                for k, v in macros.iteritems():
+                    txt = txt.replace(k, '{' + v + '}')
+            except AttributeError:
+                for k, v in macros.items():
+                    txt = txt.replace(k, '{' + v + '}')
```

Comment:
Do you think that using iteritems() in python2 is really needed here? The dictionary should probably be not that big that using items() in python 2 should be a problem. What do you think?